test/samples-js-extra/AddProductExtraColumns/function.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Let's put these under `test/Integration/test-js` so we're consistent with where we store the extra test functions. 

And then while we're doing that if you could make a `test/Integration/test-csharp` and put the C# ones there that'd be great